git-extra/PKGBUILD

```diff
@@ -65,5 +65,6 @@ package() {
   install -m755 create-shortcut.exe $pkgdir/$mingwdir/bin
   install -m755 $startdir/git-prompt.sh $pkgdir/etc/profile.d
   install -m755 $startdir/aliases.sh $pkgdir/etc/profile.d
+  install -m755 $startdir/homepath.sh $pkgdir/etc/profile.d
```

Comment:
I do not like `env` because it says even less about what purpose the file serves. The name `homepath.sh` is good enough for me.